sequencing-server/src/lib/batchLoaders/sequenceTemplateBatchLoader.ts

```diff
@@ -0,0 +1,49 @@
+import type { BatchLoader } from './index';
+import { gql, GraphQLClient } from 'graphql-request';
+import { ErrorWithStatusCode } from '../../utils/ErrorWithStatusCode.js';
+
+// TODO: grab all sequence templates for this model.
+
+export const sequenceTemplateBatchLoader: BatchLoader<
+    { modelId: number },
+    SequenceTemplate[],
+    { graphqlClient: GraphQLClient}
+> = opts => async keys => {
+  const { sequence_templates } = await opts.graphqlClient.request<{ sequence_templates: SequenceTemplate[] }>(
+      gql`
+      query GetSequenceTemplates($modelIds: [Int!]!) {
+        sequence_templates: sequence_template(where: {_and: [{ model_id: { _in: $modelIds }}]}) {
```

Comment:
Possible simplification here
```suggestion
        sequence_templates: sequence_template(where: {model_id: { _in: $modelIds }}) {
```